addon/components/custom-subsidy-form-fields/estimated-cost-table/edit.js

```diff
@@ -372,7 +382,13 @@ export default class CustomSubsidyFormFieldsEstimatedCostTableEditComponent exte
         this.updateTripleObject(this.estimatedCostTableSubject, validEstimatedCostTable, true);
       }
 
-      this.updateTripleObject(entry.estimatedCostEntrySubject, entry['cost'].predicate, entry['cost'].value);
+
+      if(isNaN(parseInt(entry.cost.value))) {
+        this.updateTripleObject(entry.estimatedCostEntrySubject, entry['cost'].predicate, "Field is empty");
```

Comment:
If value of cell NaN then make the object of that triple a string with a value. When you input something else like "", undefined, null, false and you later on look for that triple see line 345, you wont get that triple in the results and theirfor cant validate if a field was left empty

Reply:
Couldn't we set it to 0 directly if not a number ? Like ... force-set the value instead of letting it be a NaN, that would ease a bit process because we wouldn't have to check for inValidCosts ?

Reply:
Yeah I actually mentioned it to Felix, this is debatable since users wont know that an empty field is equal to 0 so I thought that this could cause confusion (especially if you are talking about money) so better to let the user put a 0 so they know what to expect